tests/plotting/test_plotting.py

```diff
@@ -2773,6 +2773,35 @@ def test_ruler():
     plotter.show()
 
 
+@skip_windows  # due to opacity
+@skip_9_0_X
```

Comment:
Forward looking: assuming these are skips due to plotting image regression only, I'm thinking that we should still run the test, but just skip the regression test.  But this test is using the current style in the test module, and it would be better to introduce a decorator (or update the existing decorators to handle this) for this suggestion.  So let's not do that here.